photon fails to start when path contains spaces

Hi,

If I start photon with java -jar photon-0.3-beta3.jar from a path which contains spaces e.g: "C:\Program Files\photon" it fails with the exception:

Exception in thread "main" java.lang.RuntimeException: Can't create directories:
        at de.komoot.photon.elasticsearch.Server.<init>(Server.java:71)
        at de.komoot.photon.elasticsearch.Server.<init>(Server.java:60)
        at de.komoot.photon.App.main(App.java:47)
Caused by: java.net.URISyntaxException: Illegal character in path at index 16: file:/C:/Program Files/photon/.
        at java.net.URI$Parser.fail(Unknown Source)
        at java.net.URI$Parser.checkChars(Unknown Source)
        at java.net.URI$Parser.parseHierarchical(Unknown Source)
        at java.net.URI$Parser.parse(Unknown Source)
        at java.net.URI.<init>(Unknown Source)
        at java.net.URL.toURI(Unknown Source)
        at de.komoot.photon.elasticsearch.Server.setupDirectories(Server.java:149)
        at de.komoot.photon.elasticsearch.Server.<init>(Server.java:66)
        ... 2 more

Tested on Windows Server 2012 R2.

Let me know if you need any additional info!

--- a/src/main/java/de/komoot/photon/elasticsearch/Server.java
+++ b/src/main/java/de/komoot/photon/elasticsearch/Server.java
@@ -6,7 +6,6 @@ import de.komoot.photon.Updater;
 import de.komoot.photon.searcher.ReverseHandler;
 import de.komoot.photon.searcher.SearchHandler;
 import lombok.extern.slf4j.Slf4j;
-import org.apache.commons.lang3.SystemUtils;
 import org.elasticsearch.action.get.GetResponse;
 import org.elasticsearch.client.Client;
 import org.elasticsearch.client.transport.TransportClient;
@@ -67,11 +66,7 @@ public class Server {
 
     public Server(String mainDirectory) {
         try {
-            if (SystemUtils.IS_OS_WINDOWS) {
-                setupDirectories(new URL("file:///" + mainDirectory));
-            } else {
-                setupDirectories(new URL("file://" + mainDirectory));
-            }
+            setupDirectories(new File(mainDirectory).toURI().toURL());
         } catch (Exception e) {
             throw new RuntimeException("Can't create directories: " + mainDirectory, e);
         }

